proto/cacheclient.proto

```diff
@@ -455,18 +455,45 @@ message _SortedSetFetchRequest {
     DESCENDING = 1;
   }
 
+  // num_results fields
   message _All {}
 
   message _Limit {
     uint32 limit = 1;
   }
 
+  // range fields
+  message _ByIndex {}
+
+  message _ByScore {
+    message _Endpoint {
+      message _Score {
+        double score = 1;
+        bool open = 2;
```

Comment:
A comment would be useful. I believe this means inclusive of the score or not?

Reply:
Added! thanks.